whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/domain/NServerValidator.java

```diff
@@ -66,4 +71,18 @@ public ImmutableList<Action> getActions() {
     public ImmutableList<ObjectType> getTypes() {
         return TYPES;
     }
+
+
+    private void validateNserverCorrectPrefixes(final Domain domain, final PreparedUpdate update,
```

Comment:
I suggest renaming to ```validateRipeNsServerPrefixLength``` to be more specific (i.e. this validation is only done for ns.ripe.net)